### ISSUE TITLE
The randomness of training results.

### ISSUE THREAD

**Issue by qqqqqqy0227** [EastbeanZhang/Gaussian-Wild#10]
Hi! Thanks for the code and paper.  I found that multiple experiments under the same experimental setup exhibit randomness. I am not sure if this is due to errors I made during the experimental process. For example, in the sacre scenario, running the same configuration and hyperparameters resulted in PSNR values of 23.24 and 22.58. 

I would like to know how to address this issue.

THANKS

**Comment by EastbeanZhang** [comment]
Due to the quality of this scene and the model learning rate, fluctuations in results indeed occur.  To achieve more stable results, try setting aside a portion of data as a validation set and save the model parameters that perform best on the validation set during training. Thanks!

**Comment by qqqqqqy0227** [comment]
Thank you for your response, I believe you are correct. I would like to know which training scenario would be more robust: Brandenburg Gate?  Trevi Fountain? or Lego? This might be important for me to explore this topic.

**Comment by EastbeanZhang** [comment]
Brandenburg Gate and Lego may be relatively better, you can have a try, thanks!

**Comment by qqqqqqy0227** [comment]
thanks
